2015-08-24  Alexandre Rostovtsev  <tetromino@gentoo.org>

        webkit2gtkinjectedbundle requires forwarding headers
        https://bugs.webkit.org/show_bug.cgi?id=148379

        Fixes GTK build with -j1

        * PlatformGTK.cmake:
 
Index: Source/WebKit2/PlatformGTK.cmake
===================================================================
--- a/Source/WebKit2/PlatformGTK.cmake	(revision 188859)
+++ b/Source/WebKit2/PlatformGTK.cmake	(working copy)
@@ -860,7 +860,7 @@ include_directories(
 )
 
 add_library(webkit2gtkinjectedbundle MODULE "${WEBKIT2_DIR}/WebProcess/gtk/WebGtkInjectedBundleMain.cpp")
-add_dependencies(webkit2gtkinjectedbundle GObjectDOMBindings)
+add_dependencies(webkit2gtkinjectedbundle GObjectDOMBindings WebKit2-forwarding-headers)
 add_webkit2_prefix_header(webkit2gtkinjectedbundle)
 
 # Add ${CMAKE_LIBRARY_OUTPUT_DIRECTORY} to LD_LIBRARY_PATH
